annotation/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>com.google.errorprone</groupId>
     <artifactId>error_prone_parent</artifactId>
-    <version>HEAD-SNAPSHOT</version>
+    <version>1.0-HEAD-SNAPSHOT</version>
```

Comment:
Module versions must begin with a number, so the dev version has been updated to `1.0-HEAD-SNAPSHOT`.